test/max_bg_threads_env_var_test.py

```diff
@@ -16,35 +17,47 @@ def min_threads(self):
+                   "('../environ_max_bg_threads_test')}")
         output, retcode = self.cmd_helper.execute_cmd(command)
         assert retcode != 1, \
-               self.fail_msg.format(f"\nError: Execution of \'{command}\' returns {retcode}. Output: {output}")
+            self.fail_msg.format(
+                f"\nError: Execution of \'{command}\'",
+                " returns {retcode}. Output: {output}")
         return output, retcode
 
     def test_TC_MEMKIND_max_bg_threads_env_var_min(self):
-        """This test checks if MEMKIND_BACKGROUND_THREAD_LIMIT environment variable limits the number of
+        """This test checks if MEMKIND_BACKGROUND_THREAD_LIMIT
+           environment variable limits the number of
            background threads spawned correctly."""
         self.threads_limit = 1
         output, retcode = self.run_test_binary()
         assert int(output) == self.min_threads, \
-               self.fail_msg.format(f"Error: There should be {self.min_threads} threads running. " \
-                                    f"Counted threads: {output}.")
+            self.fail_msg.format(
+                f"Error: There should be {self.min_threads} threads running. ",
+                f"Counted threads: {output}.")
 
     @pytest.mark.parametrize("threads_limit", [0, 10])
     def test_TC_MEMKIND_max_bg_threads_env_var_multiple_threads(self, threads_limit):
-        """This test checks if MEMKIND_BACKGROUND_THREAD_LIMIT environment variable
+        """This test checks if MEMKIND_BACKGROUND_THREAD_LIMIT
+           environment variable
            allows spawning of multiple threads."""
         output, threads_count = self.run_test_binary()
         assert int(output) > self.min_threads, \
-               self.fail_msg.format(f"Error: There should be more than {self.min_threads} threads running. " \
-                                    f"Counted threads: {output}.")
+            self.fail_msg.format(
+                f"Error: There should be more than",
+                " {self.min_threads} threads running. ",
+                f"Counted threads: {output}.")
 
     def test_TC_MEMKIND_max_bg_threads_env_var_err(self):
-        """This test checks if a negative value MEMKIND_BACKGROUND_THREAD_LIMIT environment variable is handled
+        """This test checks if a negative value
+           MEMKIND_BACKGROUND_THREAD_LIMIT environment variable is handled
            correctly."""
         self.threads_limit = -1
         output, retcode = self.run_test_binary()
         assert retcode == 134, \
-               self.fail_msg.format(f"Error: Negative value of MEMKIND_BACKGROUND_THREAD_LIMIT should not be handled. " \
-                                    f"Counted threads: {threads_count}. Output: {output}")
+            self.fail_msg.format(
+                f"Error: Negative value of MEMKIND_BACKGROUND_THREAD_LIMIT",
+                " should not be handled. ",
+                f"Counted threads: {threads_count}. Output: {output}")
```

Comment:
These changes should be a part of the previous commit. A separate commit should include a fix for this message.